src/components/2019/year-in-review/Nav.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import s from 'styled-components'
+import { Link } from 'gatsby'
+
+import { OPEN_SANS_BOLD } from '../../../utils/font'
+
+const Nav = s.div`
+    ${OPEN_SANS_BOLD}  
+    display: flex;
+    max-height: 36px;
+    background-color: #E7525F;
+`
+
+const NavBar = () => {
+  return (
+    <Nav
+      className="navbar navbar-expand-lg navbar-dark sticky-top"
+    >
+      <div className="container">
+        <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar10">
+                <span className="navbar-toggler-icon"></span>
+          </button>
+            <div className="navbar-collapse collapse" id="navbar10">
+                <ul className="navbar-nav nav-fill w-100">
+                    <li className="nav-item active">
+                        <Link className="nav-link" to="#news">NEWS</Link>
+                    </li>
+                    <li className="nav-item active">
+                        <Link className="nav-link" to="#opinion">OPINION</Link>
```

Comment:
does using the Link component actually have benefits here? You're not switching between different pages, just scrolling between different sections right? idk